package.json

```diff
@@ -12,7 +12,7 @@
   ],
   "scripts": {
     "preinstall": "npm run build",
-    "build": "fedx-scripts babel -x .js,.jsx,.ts,.tsx src --out-dir dist --ignore **/*.test.jsx,**/*.test.js,**/setupTest.js",
+    "build": "fedx-scripts babel -x .js,.jsx,.ts,.tsx src --out-dir dist --ignore **/*.test.jsx,**/*.test.js,**/setupTest.js && cp src/*.css dist/",
```

Comment:
I just tacked on a `cp` to copy the CSS file. Since we are running babel directly without webpack, there is not css-loader and didn't want to spend too much time on this.

Reply:
I think this is entirely fair since we have no need for CSS here and the future of paragon is to use CSS variables.